lib/slack-cli.rb

```diff
@@ -0,0 +1,60 @@
+require_relative "./workspace"
+
+class SlackCLI
+  def initialize(api_token)
+    @commands = {
+      'list users' => self.method(:list_users),
+      'list channels' => self.method(:list_channels),
+      'select user' => self.method(:select_user),
```

Comment:
This is a pretty advanced technique for organizing these method calls. It works, and is arguably much cleaner than a big switch statement, but it's a little concerning that you spent time figuring out how to do this but didn't complete all the waves.

In general, it is wise to do it the way you know first, then refactor it to the fancy way if you have time later. That's especially true if it's something not connected to the learning goals of the assignment.

If this code or the idea for it came from a tutor, please pass along this feedback!